keyboards/just60/rules.mk

```
@@ -0,0 +1,35 @@
+# MCU name
+MCU = atmega32u4
+
+# Processor frequency
+F_CPU = 8000000
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = qmk-dfu
```

```suggestion
BOOTLOADER = lufa-ms
```

Since you mention this PCB comes by default with a Mass Storage bootloader - I'm assuming based on the LUFA one - we should set it here. Be advised though that we don't recommend new keyboard projects use this as it is prone to bricking (keyboards that have already existed for a while, like this one, are OK though).

---

I agree that it would be better if we could built against the original bootloader. However, I forgot to backup the original flash and the merchandiser refused to send me a backup. I cannot revert to the the original bootloader to test the `lufa-ms` configuration.

---

I suggest that we keep the bootloader as `qmk-dfu`, and update to  `lufa-ms` later when someone confirms it works.

---

That's fine, `BOOTLOADER` mainly just determines the position in flash to jump to where the bootloader is, so even if it's a different bootloader and you haven't used up most of the firmware space (`lufa-ms` is 6kB and all other bootloaders are max 4kB), the `RESET` keycode will still work. You can set the correct bootloader for your board in your own keymap's rules.mk.

---

Thanks for the explanation. I did not know I could build with keymap specific rules.mk. Fixed.